server/schedule/operator/step.go

```diff
@@ -99,10 +102,16 @@ func (tl TransferLeader) Influence(opInfluence OpInfluence, region *core.RegionI
 	to.LeaderCount++
 }
 
+// Timeout returns true if the step is timeout.
+func (tl TransferLeader) Timeout(start time.Time, _ float64) bool {
```

Comment:
Cloud this method become an interface to make sure every operator have a implementation on it?

Reply:
every opstep have a implementation on it.